server/config/subject_page.proto

```diff
@@ -164,9 +164,36 @@ message ScatterTileSpec {
 }
 
 message BarTileSpec {
+  enum SortType {
+    SORT_ASCENDING = 1;
+    SORT_DESCENDING = 2;
+    SORT_ASCENDING_POPULATION = 3;
+    SORT_DESCENDING_POPULATION = 4;
+  }
   // [optional] The root url to use for labels in the x-axis. If not specified,
   // labels in the x-axis will link to Place Explorer.
   string x_label_link_root = 1;
+  // Bar height for horizontal bar charts
+  double bar_height = 2;
+  // list of specific colors to use
```

Comment:
nit: What are the color values, just english words, small-case?

Reply:
Any CSS-compatible color string (hex, rgb, rgba, etc) is allowed. Updated the comment line here and everywhere else `colors` shows up in the file.